featuretools/tests/primitive_tests/aggregation_primitive_tests/test_agg_primitives.py

```diff
@@ -239,11 +239,13 @@ def test_regular(self, dtype):
         given_answer = primitive_func(data)
         assert np.isclose(answer, given_answer, atol=0.01)
 
-        data = pd.Series([sqrt(x) for x in list(range(100))], dtype=dtype)
-        answer = -0.46
-        primitive_func = self.primitive().get_function()
-        given_answer = primitive_func(data)
-        assert np.isclose(answer, given_answer, atol=0.01)
+        if dtype == "float64":
```

Comment:
This is not a blocker (more of an incidental comment), but seeing the `if` clause here makes me consider if we should eventually refactor this test. Given that the previous `data` objects are being cast to `float64` dtype as well, and the test does not require `if` clauses, it seems like there are three separate classes of behaviors here: 

- How the kurtosis method handles int64 dtype
- How the kurtosis primitive handles float64 dtype when the data "looks like" an int64 (evidently this doesn't seem to differ from the previous class) 
- How the kurtosis primitive handles float64 dtypes when the data is a "true" float 

Separating those might help to better document this primitive's behavior. In any case, this is probably something to revisit later in a future issue.